### PR TITLE
#587 new resource search bar UI implementation

### PR DESCRIPTION
**Pull request by Xerxi88**

### CODE REVIEW

**Review by JuditKaramazov — APPROVED** [review]

**Review by NataliaR-BCN — CHANGES_REQUESTED** [review]
Please, translate the added texts.
And maybe consider refactoring the code into a separate component.

File: front/src/pages/Category.tsx [hunk 1]
```diff
@@ -502,30 +584,94 @@ const Category: FC = () => {
...
+                    {t('Filtrar')}
                   </FilterButton>
                 </TitleResourcesContainer>
-                <VotesDateController
-                  sortOrder={sortOrder}
-                  handleSortOrder={handleSortOrder}
-                  handleSortByVotes={handleSortByVotes}
-                  handleSortByDates={handleSortByDates}
-                />
+                <VotesDateContainer>
```
Comment (JuditKaramazov) [inline front/src/pages/Category.tsx:633]: Great job so far! However, I noticed that the recent changes introducing 'VotesDateController' were reverted, and 'VotesDateContainer' is back in use. Please note that 'VotesDateController' contains all the previous logic and styles while staying as an isolated component now!

**Review by JuditKaramazov — CHANGES_REQUESTED** [review]

File: front/CHANGELOG.md [hunk 2]
```diff
@@ -2,8 +2,12 @@
 
 All notable changes to this project will be documented in this file.
 
-## [0.2.1] - 2023-11-16
+## [0.3.1] - 2023-11-16
```
Comment (kevinmamaqi) [inline front/CHANGELOG.md:5]:
```suggestion
## [0.3.0] - 2023-11-16
```

File: front/package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "front",
   "private": true,
-  "version": "0.2.1",
+  "version": "0.3.1",
```
Comment (kevinmamaqi) [inline front/package.json:4]:
```suggestion
  "version": "0.3.0",
```

**Review by kevinmamaqi — CHANGES_REQUESTED** [review]

**Pull request closed without merging**